README.md

```diff
@@ -26,37 +26,38 @@ An [OpenAPI specification](https://github.com/OAI/OpenAPI-Specification) describ
-| [Look#](csharp)            | `Looker31SDK()`                                                           | `Looker40SDK()`                                                          | Community-supported C# SDK for Looker                                                                                                                                                                                                          |
-| [GoLook](go)               | Not applicable                                                            | `v4.NewLookerSDK()`                                                      | Community-supported GO SDK for Looker                                                                                                                                                                                                          |
+| SDK                        | API 3.1 [(deprecated)](https://developers.looker.com/api/advanced-usage/version-3x-deprecation) | API 4.0                                                                  | Notes                                                                                                                                                                                                                                          |
+| -------------------------- | ----------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Python](python)           | `looker_sdk.init31()`                                                                           | `looker_sdk.init40()`                                                    |                                                                                                                                                            |
+| [TypeScript](packages/sdk) | `Looker31SDK()`, `LookerNodeSDK.init31()`, or `LookerBrowserSDK.init31()`                       | `Looker40SDK()`, `LookerNodeSDK.init40()` or `LookerBrowserSDK.init40()` | **Important** - See information on the [typescript SDK dependencies](#very-important-note-regarding-the-looker-typescript-sdk) at the bottom of this file. |
+| [Kotlin](kotlin)           | Do not use                                                                                      | `LookerSDK()`                                                            | API 4.0 was created to better serve strongly-typed languages, like Kotlin. Kotlin only works with API 4.0. Also, the initializer uses an unversioned name. |
```

Comment:
If 3.1 is gone from Kotlin, you can make it "Not applicable" as well.

Reply:
Updated all strongly typed SDKs to 3.1 "Not supported"